src/v/redpanda/admin/server.cc

```diff
@@ -1698,6 +1698,28 @@ void config_multi_property_validation(
         errors[ss::sstring(name)] = ssx::sformat(
```

Comment:
> admin/server: add cloud_storage_cache_size check...
> ...to `config_multi_property_validation()`

please try to keep the length of the commit subject line small enough that it doesn't wrap

Reply:
I wrapped it manually, thinking that verbosity was preferred over style, but well noted. I have altered the commit message.